Yoast/yoast-components
# Stories/249 wizard css tweaks

## Summary

This PR can be summarized in the following changelog entry:

* Onboarding Wizard styling changes

## Test instructions

This PR can be tested by following these steps:

* Open the onboarding wizard
* Check if the CSS changes described in the issue have been applied

Fixes #249
